src/main/java/edu/ucsb/cs156/gauchoride/entities/DriverAvailability.java

```diff
@@ -0,0 +1,31 @@
+package edu.ucsb.cs156.gauchoride.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "driveravailability")
+public class DriverAvailability {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    private String driverId;
```

Comment:
I think, to be consistent with how shifts are implemented, we should use a long for the driverId instead of a string.

When I was writing the issue, I wasn't sure, but now I think it should be a long

Reply:
changed from string to long in the lastest commit.